package.json

```diff
@@ -14,11 +14,14 @@
   },
   "homepage": "https://github.com/adrianmcli/truffle-react#readme",
   "dependencies": {
+    "chai": "^4.1.2",
+    "chai-bignumber": "^2.0.2",
```

Comment:
It doesn't look like we are using `chai` nor `chai-bignumber`. Please remove?

Reply:
removed